Source/Core/DolphinWX/NetPlay/NetPlaySetupFrame.cpp

```diff
@@ -19,40 +19,17 @@
 #include "Core/NetPlayServer.h"
 #include "DolphinWX/Frame.h"
 #include "DolphinWX/Main.h"
+#include "DolphinWX/NetPlay/NetPlayLauncher.h"
 #include "DolphinWX/NetPlay/NetPlaySetupFrame.h"
 #include "DolphinWX/NetPlay/NetWindow.h"
 #include "DolphinWX/WxUtils.h"
 
 namespace
```

Comment:
Considering it's not set in stone only one internal function will exist, yes. The entire point of an anonymous namespace is to provide an area for internal linkage for potential re-use. It also has benefits over static if a linker error occurs involving something inside of it. You'll get `(anonymous namespace)::some_func()` rather than an ambiguous `some_func()` like you would with static, in the error output which makes it easier to identify